README.md

```diff
@@ -5,7 +5,7 @@ WorldEngine - a world generator
 
 _The current stable version is 0.18.0_
 
-You can generate worlds data (heighmap, biome, etc.) and images for your own worlds.
+You can generate your own world, including images (heigthmap, biomes, etc.).
```

Comment:
I would underline that there is data produced: the images are a showcase to lure the user but the product is the data imho